packages/wrangler/src/__tests__/api-dev.test.ts

```diff
@@ -16,4 +21,192 @@ describe("unstable_dev", () => {
 		}
 		await worker.stop();
 	});
+
+	describe("protocol", () => {
+		it("should use https localProtocol", async () => {
+			const port = 8080;
+			msw.use(
```

Comment:
You don't need to use MSW here btw - this test uses an actual fetch against a running worker

Reply:
It was some hacky stuff I was trying. Dont worry.